tests/ddos/ddo_sample_algorithm_v4.py

```diff
@@ -60,6 +61,7 @@
                 "publisherTrustedAlgorithmPublishers": [],
             },
             "files": "encryptedFiles",
+            "consumerParameters": {"name": "test_key"},
```

Comment:
Can you add an example that shows `consumerParameters` can be a list of multiple `ConsumerParameter` instances?